lib/tasks/generate_elo_csv.rake

```diff
@@ -53,29 +53,13 @@ task generate_elo_csv: :environment do
     [row["Game-ID"], parse_date(row["Date"]), row["Winner"], match_player]
   end
 
-  def update_sigma_decay(ts_rating, weeks_since_last_played)
```

Comment:
Why is this necessary?

Reply:
Short answer is it messes up the calculation for subsequent elo's and has given wild swings up and down to peoples elo's, better off asking Stormtrout for details